src/services/WolfDen.Application/Requests/Queries/Attendence/SubOrdinates/TeamHeirarchyService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore;
+using WolfDen.Application.DTOs.Attendence;
+using WolfDen.Domain.Entity;
+using WolfDen.Infrastructure.Data;
+
+namespace WolfDen.Application.Requests.Services
+{
+    public class TeamHeirarchyService(WolfDenContext context)
+    {
+        private readonly WolfDenContext _context = context;
+        public async Task<List<SubOrdinateDTO>> GetSubordinates(int managerId)
+        {
+            List<SubOrdinateDTO> result = new List<SubOrdinateDTO>();
+            List<Employee> employees = await _context.Employees.Where(x => x.ManagerId == managerId && x.IsActive == true).Include(x=>x.Department).Include(x=>x.Designation).ToListAsync();

```

Comment:
x.IsActive , no need to equate it to true

Reply:
but it is a nullable field

Reply:
okay understood